### PR TITLE
21194 Added handling for pending filing with completed payment

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#21194

*Description of changes:*
- app version = 7.2.14
- added pending filing sub-title when payment is completed
- do not show actions when filing is pending but payment is completed
- show PaymentPaid sub-component when payment is completed
- added contact info to PaymentPaid sub-component

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

---

before

after

### CODE REVIEW

File: src/components/Dashboard/TodoList.vue [hunk 1]
```diff
@@ -122,9 +122,16 @@
...
+                  v-if="isAffiliationInvitation(item)"
+                  class="todo-subtitle"
+                >
+                  <span v-if="!!item.subtitle">{{ item.subtitle }}</span>
+                </div>
+
                 <!-- new todo task -->
                 <div
-                  v-if="isStatusNew(item) || isAffiliationInvitation(item)"
+                  v-else-if="isStatusNew(item)"
```
Comment (severinbeauvais) [inline src/components/Dashboard/TodoList.vue:134]: I split up the affiliation-invitation and new-todo-task cases. This works better with code-folding and de-couples the cases for future refactoring.

File: src/components/Dashboard/TodoList.vue [hunk 2]
```diff
@@ -1159,7 +1168,8 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
         order: task.order,
         paymentMethod: header.paymentMethod || null,
         paymentToken: header.paymentToken || null,
-        payErrorObj
+        payErrorObj,
+        isPayCompleted: (paymentStatusCode === 'COMPLETED')
```
Comment (severinbeauvais) [inline src/components/Dashboard/TodoList.vue:1172]: No, we don't have an enum for Payment Status Code.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-665-dovqwyal.web.app

**Review by ketaki-deodhar — APPROVED** [review]
Looks good. Just a small observation (hope it's not just me that see this :D). Word "Paid" looks like "Pald". Probably not much we can do.
image

**Comment by severinbeauvais** [comment]
> Looks good. Just a small observation (hope it's not just me that see this :D). Word "Paid" looks like "Pald". Probably not much we can do.

I see it, too. It's a font issue. Here it is a bit larger:

image

This has been happening for several years. Yui never pursued it 🤷‍♂️

**Comment by ketaki-deodhar** [comment]
> > Looks good. Just a small observation (hope it's not just me that see this :D). Word "Paid" looks like "Pald". Probably not much we can do.
> 
> I see it, too. It's a font issue. Here it is a bit larger:
> 
> image
> 
> This has been happening for several years. Yui never pursued it 🤷‍♂️

Yes. Looks okay in larger font.

**Review by JazzarKarim — APPROVED** [review]
Looks good Sev 👍

**Pull request merged**